### DISCUSSION TITLE
Update Wikis' Markdown Support

### DISCUSSION THREAD

**Discussion by StormFX** [community/community#6195] [Other Feature Feedback, Questions, & Ideas]
There have been some nice additions to the GitHub-Flavored Markdown across the majority of GitHub, however, the wikis don't seem to be included in these updates. Some examples include the auto-generated ToC menus at the top of markdown files and the recently-added footnotes feature.

While I'm aware that wikis now generate nested ToCs in the sidebar, this requires that the wiki authors rely on the page list, which doesn't always provide the best navigational flow.

It would be nice to see these features added to the wiki engine. Thanks! :)

**Comment by samwar** [comment]
+💯  I was surprised to see the markdown footnotes feature didn't work for the wiki. It would be really nice if the rendering features worked across all of places in github where text is rendered.

**Reply by StormFX** [reply]
Agreed, The GitHub-flavored markdown structure should be consistent across the platform. The wiki is lagging behind in this, and updates in general. :/

**Comment by numist** [comment]
I found this discussion after being referred to the community discussions from support [Ticket 2156437](https://support.github.com/ticket/2156437). Would love to see footnote (and GFM more broadly) support in wikis!

**Comment by igorsantos07** [comment]
This is a very, very, very old request by hundreds of users:
- https://github.com/isaacs/github/issues/215 - the README ToC seems to have been eventually solved
- https://github.com/isaacs/github/issues/504 but it looks like everyone forgot about the wiki.

It makes no sense having a separate Markdown engine for rendering the repo files and the wiki... does it? Isn't it all markdown in a git repo?

The current ToC on the sidebar is mostly useless, given:
1. it's not shown by default when you open the related wiki page, thus the user needs to _decide to look for headings_ - but instead they'll probably just scroll or ctrl+F
2. it doesn't stay fixed when scrolling, so you need to go to the top whenever you want to change to another section. It's the same behavior we get from a manual ToC (or an automatic one from AsciiDoc)
3. the current page's ToC doesn't get more prominent from the sidebar, which is actually more like a page navigator than an actual table of contents.

So far, the only solution we found was to migrate the bigger pages from our wiki to AsciiDoc, but its syntax is unusual, to say the least, and ugly at some aspects.
